packages/fast-components-react-base/src/text-field/text-field.props.ts

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { IManagedClasses, ITextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export enum TextFieldType {
+   date = "date",
```

Comment:
@nicholasrice do we want to do it this way?

Reply:
The enum looks right to me, is that what you're reffering to?